### ISSUE TITLE
Entities only update once during installation

### ISSUE THREAD

**Issue by CyberrNerd** [oyvindwe/connectlife-ha#2]
As mentioned in the title, after installation I do see all entities of my dishwasher, also with the right values. But after the installation the values are not updated anymore. What can be wrong? 

In the error log I see something like:

[SSL: SSLV3_ALERT_BAD_RECORD_MAC

I really like this addon and really hope entities can be controlled as well in the near future! Great job!

**Comment by Monacoslo** [comment]
I have noticed similar issue. But also restart of HA updates the entities status. They just do not have live update

**Comment by deb0ro** [comment]
I see the same behavior on supervised HA running on x86.

**Comment by oyvindwe** [comment]
This is very hard to debug, as I don't get this error. I occessionally get 
```
ERROR (MainThread) [custom_components.connectlife.coordinator] Error fetching ConnectLife data: Error communicating with API: Unexpected response: status=500
```
but in my case the component resumes working on the next request in these cases.

I've done some research on the error message, and firstly it's important to notice that `SSLv3` is a bit misleading, as this was the current protocol when the error message was introduced in OpenSSL - it may apply to newer protocol version as well. I found some advise that this may happen if clocks are out of sync, so I suggest you check your system clocks. Allthough since it happens for multiple users suggest it's a problem on the ConnectLife API side.

Also which version of the component are you running? I added some improved error handling in version 0.0.5, but not addressing this issue specifically.

Can someone provide the entire stacktrace from the log? Maybe it's an error that can be retried.

**Comment by CyberrNerd** [comment]
Now using the latest version 0.0.5. I do see this in the log:

2024-05-25 19:33:05.427 DEBUG (MainThread) [custom_components.connectlife.coordinator] Finished fetching ConnectLife data in 0.201 seconds (success: True)

It shows this message each minute, but it doesnt update my entities?

**Comment by CyberrNerd** [comment]
I also see this error:
**Error fetching ConnectLife data: Error communicating with API: Unexpected response: status=500**

But as you mentioned, after this it seems I just get these messages each minute again:
**2024-05-25 20:53:08.432 DEBUG (MainThread) [custom_components.connectlife.coordinator] Finished fetching ConnectLife data in 0.207 seconds (success: True)**

But no update of entities at all...

**Comment by oyvindwe** [comment]
Hmm - I think I have the same problem now! Then I should be able to debug.

**Comment by oyvindwe** [comment]
I finally found some time to figure this out. Should be fixed in version 0.0.6
